src/server/game/Entities/Creature/Creature.h

```diff
@@ -590,6 +590,7 @@ class TC_GAME_API Creature : public Unit, public GridObject<Creature>, public Ma
 
         bool isCanInteractWithBattleMaster(Player* player, bool msg) const;
         bool CanResetTalents(Player* player) const;
+        bool CanCreatureAttack(Unit const* victim, bool force = true) const;
```

Comment:
We already have _CanCreatureAttack which is almost the same one. It has been renamed because if has a different return type. You can just create a CanCreatureAttack that calls to that one for compat.
(Just add a CanCreatureAttack that calls to _CanCreatureAttack)

Reply:
Added compat function